manifests/dm/README.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
move docs to `docs/` directory?

Reply:
This is just for temp deployment, we may need to support DM with chart or controller later, I think we can leave it here for now.